test/dummyspec/streetlights.yml

```diff
@@ -0,0 +1,209 @@
+asyncapi: '2.0.0'
```

Comment:
as this is only for the tests related to fetching schema from URL, let's make it super super dummy minimalistic file, so please rename to `dummyFetchTest.yml` and content like
```
asyncapi: 2.6.0
info:
  title: Dummy example used in tests related to fetching AsyncAPI from URL
  version: 0.0.1
channels: {}
```

Reply:
@chinma-yyy yo, I think you forgot about this one

Reply:
No, actually the `convert` command tests depend on this older version spec. So,if the version is changed the test suite will fail :v:

Reply:
then change to `2.5`. More important is to have dummy, non streetlight example